syncd/SaiSwitchInterface.h

```diff
@@ -89,7 +89,8 @@ namespace syncd
 
             virtual void onPostPortCreate(
                     _In_ sai_object_id_t port_rid,
-                    _In_ sai_object_id_t port_vid) = 0;
+                    _In_ sai_object_id_t port_vid,
+                    _In_ bool discoverPortObjects = true) = 0;
```

Comment:
this is very strict to ports, if we decide later on to do something similar on other objects then this is not optimal solution

Reply:
The function is meant to be used on ports. Considering current approach, I assume there will be onPostXCreate() functions for other object types. Then, if needed, they can accept a boolean flag in the same way. This is simple and gives required granularity.